CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing
+
+Contributions are **welcome** and will be fully **credited**.
+
+We accept contributions via Pull Requests on [Github](https://github.com/yajra/{{package}}).
```

Comment:
Please fix {{package}} with the actual package name. Thanks!

Reply:
Change link applied!